content-repo/extra-docs/articles/CI-CD-Article.md

```diff
@@ -446,6 +467,14 @@ Although you do not have the flexibility of version control and rollback, it is
    ![cicd_playbook.png](../../../docs/doc_imgs/reference/XSOAR-CICD/cicd_playbook.png)
   
    - If not using the XSOAR CI/CD Content Pack, install content manually. For example, if using Google Cloud Services integration run the `gcd-download-file` command.
+  
+8. If you have any detached content which you have downloaded in step 4 in Setup the CI/CD Development Process above, run the following command:
+   `sdk upload --input-config-file <path to download the all the system packs from>` 
+   
+   For example, `sdk upload --input-config-file SystemPacks/Phishing/Playbooks.`
+
+   **NOTE**: When you upload detached content to Cortex XSOAR it overrides any existing attached/detached content. To keep any changes make a duplicate of the existing detached content in Cortex XSOAR before uploading.
+
```

Comment:
If there is content in the SystemPacks folder the `sdk upload --input-config-file` command will detach all this content automatically (no need to specify the path).

Regarding the reattach - I've just added a new arg `reattach`, if the user will run the command with this arg we will compare the repo aginst the instance - just the files that exist in the SystemPacks will remain detached and all other items in the instance will reattach.

Reply:
Updated the paragraph